.github/workflows/ci.generate.ts

```diff
@@ -812,9 +812,7 @@ const ci = {
             "github.ref == 'refs/heads/main'",
           ].join("\n"),
           run: [
-            'gsutil -h "Cache-Control: public, max-age=3600" cp ./target/release/*.zip gs://dl.deno.land/canary/$(git rev-parse HEAD)/',
-            "echo ${{ github.sha }} > canary-latest.txt",
-            'gsutil -h "Cache-Control: no-cache" cp canary-latest.txt gs://dl.deno.land/canary-$(rustc -vV | sed -n "s|host: ||p")-latest.txt',
+            "./tools/release/upload_canary.ts arch ${{ github.sha }}",
```

Comment:
There's been some changes on main since this PR to add the shasums and symcache:

https://github.com/denoland/deno/blob/3071aa2759a6092fcd56b06eee63303aa3425df0/.github/workflows/ci.generate.ts#L863-L867